frontend/views/containers/user-settings/Appearence.vue

```diff
@@ -33,6 +33,20 @@
             :checked='$store.state.reducedMotion'
             @change='handleReducedMotion'
           )
+
+      i18n.is-title-2.c-title(tag='h2') Mute chat
+      .c-subcontent
+        .c-text-content
+          i18n.c-smaller-title(tag='h3') Blablablabla
+          i18n.help(tag='p') Blablabla blablabla blablabla blablabla blablabla
+        label
+          i18n.sr-only Blablablabla
```

Comment:
What's going on here...?

Reply:
It's just saving the info to mute chat or not. Just save it in state but not implemented function yet.

Reply:
OK, we cannot have "blahblahblah" going into `master`, that is not professional :P

Reply:
I will remove this. I think that's true also. That's because it's not the one from the design.